source/AI.cpp

```diff
@@ -3629,7 +3645,7 @@ void AI::MovePlayer(Ship &ship, const PlayerInfo &player, Command &activeCommand
 		{
 			PrepareForHyperspace(ship, command);
 			command |= Command::JUMP;
-			
+
```

Comment:
Removing this is fine; though generally you only want to touch lines of code where you're changing logic.

Reply:
I think this was added by a miss-merge, all code files should be free of useless whitespaces. 
This PR is so minuscule that I don't mind it this time, but samenrocketman is right.